hoomd/md/pytest/test_pair.py

```diff
@@ -218,6 +218,10 @@ def _calculate_force(sim):
 def test_force_energy_relationship(simulation_factory,
                                    two_particle_snapshot_factory,
                                    valid_params, nsteps):
+    # don't really test DPD and DPDLJ for this test
+    if valid_params[0] == "DPD" or valid_params[0] == "DPDLJ":
+        return
```

Comment:
Skip the test instead of returning: https://docs.pytest.org/en/stable/skipping.html